management/src/main/java/org/ehcache/management/providers/statistics/StandardEhcacheStatistics.java

```diff
@@ -195,7 +200,9 @@
       } else if(registeredStatistic instanceof RegisteredRatioStatistic) {
         capabilities.add(new StatisticDescriptor(entry.getKey() + "Ratio", StatisticType.RATIO_HISTORY));
       } else if(registeredStatistic instanceof RegisteredValueStatistic) {
-        capabilities.add(new StatisticDescriptor(entry.getKey().toString(), StatisticType.COUNTER_HISTORY));
+        capabilities.add(new StatisticDescriptor(statisticName, statisticName.endsWith("Size") ?
```

Comment:
How much code is it? I think that relying on `String` values to drive the difference is something that will cost in the future ...